nipype/interfaces/fsl/epi.py

```diff
@@ -523,11 +521,27 @@ def _num_threads_update(self):
             self.inputs.environ['OMP_NUM_THREADS'] = str(
                 self.inputs.num_threads)
 
-    def _use_cuda(self):
-        if self.inputs.use_cuda:
-            _cmd = 'eddy_cuda'
-        else:
-            _cmd = 'eddy_openmp'
+    @staticmethod
+    def _eddycmd(use_cuda):
+        logger = logging.getLogger('interface')
+        if 'FSLDIR' not in os.environ:
+            logger.warn("FSLDIR not set: assuming command 'eddy'")
+            return 'eddy'
+
+        FSLDIR = os.environ['FSLDIR']
+        if use_cuda and os.path.exists(os.path.join(FSLDIR, 'eddy_cuda')):
```

Comment:
i see how this is problematic for a lot of things, but the only thing i was hoping for was to replace `eddy_openmp` with `eddy` on systems that have eddy but not eddy_openmp at runtime. (so in an overloaded run_interface call.

the previous deleted `_use_cuda` seems reasonable here and can also be used in doctests without requiring fsl to be present.